src/BookTable/style.scss

```diff
@@ -0,0 +1,124 @@
+@import '../variables/_variables.scss';
+
+.book-wrapper {
+  background: fixed url('../image/form-bg.png') no-repeat center center / cover;
+}
+
+.book-form {
+  display: flex;
+  justify-content: center;
+  padding-bottom: 30px;
+
+  @media screen and (max-width: $max-sm-screen) {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding-bottom: 30px;
+    margin: 0 auto;
+  }
+}
+
+.left-side {
+  @media screen and (max-width: $max-md-screen) {
+    align-items: center;
+  }
+
+  @media screen and (max-width: $max-sm-screen) {
+    max-width: 50%;
+
+  }
+
+  @media screen and (max-width: $max-xs-screen) {
+    display: none;
+  }
+  display: flex;
```

Comment:
main styles should be before media queries

Reply:
done